configure.ac

```diff
@@ -541,507 +540,17 @@ AC_ARG_WITH([rdp],
-
-                                        int main() {
-                                            BOOL* flip = TRUE;
-                                            planar_decompress(NULL, NULL, 0, NULL, 0, 0, 0, 0, 0, 0, flip);
-                                        }]])],
-                      [AC_MSG_RESULT([yes])
-                       AC_DEFINE([PLANAR_DECOMPRESS_CAN_FLIP],,
-                                 [Whether planar_decompress() can flip])],
-                      [AC_MSG_RESULT([no])])
-fi
-
-#
-# FreeRDP: rdpContext
-#
-
-# Check for rdpContext.codecs
-if test "x${have_freerdp}" = "xyes"
-then
-    AC_CHECK_MEMBERS([rdpContext.codecs],
-                     [AC_DEFINE([FREERDP_BITMAP_REQUIRES_ALIGNED_MALLOC],,
-                                [Whether this version of FreeRDP requires _aligned_malloc() for bitmap data])],,
-                     [[#include <freerdp/freerdp.h>]])
-fi
-
-#
-# FreeRDP: rdpPalette
-#
-
-if test "x${have_freerdp}" = "xyes"
-then
-    AC_MSG_CHECKING([whether rdpPalette.entries is static])
-    AC_COMPILE_IFELSE([AC_LANG_SOURCE([[#include <freerdp/update.h>
-                                        rdpPalette p;
-                                        PALETTE_ENTRY* foo = p.entries;]])],
-                      [AC_MSG_RESULT([yes])],
-                      [AC_MSG_RESULT([no])
-                       AC_DEFINE([LEGACY_RDPPALETTE],,
-                                 [Whether the legacy rdpPalette API was found])])
-fi
-
-#
-# FreeRDP: rdpPointer
-#
-
-# Check for SetDefault and SetNull members of rdpPointer
-if test "x${have_freerdp}" = "xyes"
-then
-    AC_CHECK_MEMBERS([rdpPointer.SetDefault,
-                      rdpPointer.SetNull],
-                     ,,
-                     [[#include <freerdp/freerdp.h>]])
-fi
-
-#
-# FreeRDP: wMessage / RDP_EVENT
-#
-
-# Check for current (as of 1.1) wMessage interface
-if test "x${have_freerdp}" = "xyes"
-then
-    AC_CHECK_MEMBERS([wMessage.id],
-                     [event_interface=stable],,
-                     [[#include <winpr/collections.h>]])
-
-    # If not current, check for legacy (RDP_EVENT) interface
-    if test "x${event_interface}" = "xunknown"
-    then
-        AC_CHECK_MEMBERS([RDP_EVENT.event_class],
-                         [event_interface=legacy],,
-                         [[#include <freerdp/types.h>]])
-    fi
-
-    # Set defines based on interface type, warn if unknown
-    if test "x${event_interface}" = "xlegacy"; then
-        AC_DEFINE([LEGACY_EVENT],,
-                  [Whether the legacy RDP_EVENT API was found])
-    elif test "x${event_interface}" = "xunknown"; then
-        AC_MSG_WARN([
-  --------------------------------------------
-   Unknown event interface
-   RDP will be disabled.
-  --------------------------------------------])
-        have_freerdp=no
-    fi
-fi
-
-AM_CONDITIONAL([LEGACY_FREERDP_EXTENSIONS], [test "x${legacy_freerdp_extensions}" = "xyes"])
```

Comment:
It's pretty awesome that all this stuff is going away!

Reply:
Yup!